### DISCUSSION TITLE
如何在非go-zero的项目中，只使用core包的核心工具呢

### DISCUSSION THREAD

**Discussion by lingyurunxin** [zeromicro/go-zero#2051] [Q&A] [answered]
本人在开发一个非go-zero项目，感觉go-zero的core包质量非常高，想通过go get的方式，引入到自己的项目中，如何实现呢

**Comment by kevwan** [comment]
你引用就好了，只是使用core包即可。

**Reply by lingyurunxin** [reply]
作为java转go初学者，不太清楚如何只引入core，是直接copy core包到项目中吗(感觉不是很优雅，希望通过go get的方式)？

**Comment by kevwan** [comment, marked as answer]
go get github.com/zeromicro/go-zero

**Reply by lingyurunxin** [reply]
这样不就将整个go-zero都引入项目了吗，不会带来项目冗余吗？😂

**Comment by kevwan** [comment]
go编译器会自动剔除冗余代码的。

**Reply by lingyurunxin** [reply]
谢谢Kevin👍
